contracts/CountTransferManager/README.md

```diff
@@ -1,23 +1,29 @@
-# polymesh-count-transfer-manager
+# CountTransferManager
+This smart contract is a transfer manager for limiting the number of token holders a ticker can have.
+  
 
-Count Transfer Manager implemented in Ink! smart contract.
+## Pre-requsite
+`cargo-contract` Install using below command
+```
+cargo install --git https://github.com/paritytech/cargo-contract cargo-contract --force
+```
 
-Designed to be attached to the Asset module to allow the total number of investors to be capped.
+### Build example contract and generate the contracts metadata
 
-# Pre-Requisites
+To build a single example and generate the contracts Wasm file, navigate to the root of the example smart contract and run:
 
-  - Rust (nightly and stable)
-  - Contract Crate
-  `cargo install --force --git https://github.com/paritytech/ink cargo-contract`
+```
+cargo +nightly contract build
```

Comment:
Is it any nightly? Isn't the compiler version already defined in `./rust-toolchain`?

Reply:
That change wasn't included when I created this PR. Anyway, even right now It will not affect as the workspace for contracts is different. I haven't included that in our workspace to avoid the increase if the size and time of build as the idea is to keep contracts not dependent on the chain directly.

Reply:
Somehow it starts working without `+nightly`